frontend/src/features/workspace/components/WorkspacePageContent.tsx

```diff
@@ -11,8 +11,6 @@ import { useUser } from '~features/user/queries'
 import { useWorkspaceContext } from '../WorkspaceContext'
```

Comment:
actually what was this one about ah HAHA i thought it was the rollout announcement modal, do we still want to keep the rollout announcement? if it's a permanent fixture then i'll delete the comment, otherwise we can open another issue to delete the rollout announcement modal

Reply:
honestly I have no idea.. maybe it was an accidental additional LOL. I think we still want to keep the rollout announcement modal since it's announcements about new features! @staceytan1998 can confirm?

Reply:
Confirmed with stacey that we can delete the rollout announcement also! will tag that as a separate issue.